From d93e4dcc7388511ee3100bf481683c9bbdfdd593 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Mon, 23 Jun 2025 20:31:20 +0800
Subject: [PATCH 17/63] loonggpu: Remove a redundant gsgpu_fbdev_client_hotplug
 call

It's not needed anymore as now the generic drm_client_register() calls
the hotplug hook.

Link: https://git.kernel.org/torvalds/c/27655b9bb9f0
Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 loonggpu/loonggpu_fb.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/loonggpu/loonggpu_fb.c b/loonggpu/loonggpu_fb.c
index 7f40eae..e225499 100644
--- a/loonggpu/loonggpu_fb.c
+++ b/loonggpu/loonggpu_fb.c
@@ -361,10 +361,6 @@ void loonggpu_fbdev_setup(struct loonggpu_device *adev)
 		goto err_drm_client_init;
 	}
 
-	ret = loonggpu_fbdev_client_hotplug(&fb_helper->client);
-	if (ret)
-		drm_dbg_kms(adev->ddev, "client hotplug ret = %d\n", ret);
-
 	drm_client_register(&fb_helper->client);
 
 	return;
-- 
2.52.0

